docs/reference/contributing/guidelines/ci.md

```diff
@@ -0,0 +1,56 @@
+<h2 id="ci">Continuous integration</h2>
+
+Continuous integration (CI) means mainly automatic testing for pull requests.
+
+### Travis CI
+
+In Mbed OS [Travis CI](https://travis-ci.org/ARMmbed/mbed-os) is used as primary automatic testing and checking run environment.
+
+Travis configuration is located in the [.travis.yml](https://github.com/ARMmbed/mbed-os/blob/master/.travis.yml) file in Mbed OS root directory. Mbed OS uses public travis so test results are publicly available and there are public [documentation available](https://docs.travis-ci.com/).
+
+#### Tests
+
+- **continuous-integration/travis-ci/pr** - Main run main.
```

Comment:
What does this mean?

Reply:
I'm not sure how to clarify this. Could someone please explain what this means?

Reply:
This job is the main job that runs all of the other travis-ci sub-jobs. Not sure what kind of wording would work for this.

Reply:
"Run other testing jobs."?

Reply:
Does this phrasing work?

Reply:
@OPpuolitaival Do you think you could clarify this? Looks like it should be the last thing for this PR.

Reply:
I went with the phrasing I suggested. If you have another suggestion, please put up a new PR.